app/transformations/UserAnswersToETMP/API1830.scala

```diff
@@ -63,110 +63,122 @@ object API1830 extends Transformer {
+      ).reduce
   }
 
   private def readsIndividualMemberDetailsEvent7: Reads[JsObject] = {
+
     (
-      (__ \ Symbol("eventType")).json.put(JsString(s"Event$Event7")) and
+      (pathToEvent \ Symbol("eventType")).json.put(JsString(s"Event$Event7")) and
         readsIndividualMemberDetails and
         (pathPaymentDetails \ Symbol("amountLumpSum")).json.copyFrom((__ \ Symbol("lumpSumAmount")).json.pick) and
         (pathPaymentDetails \ Symbol("amountCrystalised")).json.copyFrom((__ \ Symbol("crystallisedAmount")).json.pick) and
-        (pathPaymentDetails \ Symbol("eventDate")).json.copyFrom((__ \ Symbol("paymentDate") \ Symbol("date")).json.pick)).reduce
+        (pathPaymentDetails \ Symbol("eventDate")).json.copyFrom((__ \ Symbol("paymentDate") \ Symbol("date")).json.pick) and
+        pathMemberStatus.json.put(memberStatusNew)
+      ).reduce
   }
 
   private def readsIndividualMemberDetailsEvent8: Reads[JsObject] = {
     (
-      (__ \ Symbol("eventType")).json.put(JsString(s"Event$Event8")) and
+      (pathToEvent \ Symbol("eventType")).json.put(JsString(s"Event$Event8")) and
         readsIndividualMemberDetails and
         (pathPaymentDetails \ Symbol("amountLumpSum")).json.copyFrom((pathLumpSumAmountAndDateDetails \ Symbol("lumpSumAmount")).json.pick) and
         (pathPaymentDetails \ Symbol("typeOfProtection")).json.copyFrom(readsTypeOfProtectionEvent8) and
         (pathPaymentDetails \ Symbol("eventDate")).json.copyFrom((pathLumpSumAmountAndDateDetails \ Symbol("lumpSumDate")).json.pick) and
-        (pathPaymentDetails \ Symbol("freeText")).json.copyFrom((__ \ Symbol("typeOfProtectionReference")).json.pick)).reduce
+        (pathPaymentDetails \ Symbol("freeText")).json.copyFrom((__ \ Symbol("typeOfProtectionReference")).json.pick) and
+        pathMemberStatus.json.put(memberStatusNew)
+      ).reduce
   }
 
   private def readsIndividualMemberDetailsEvent8A: Reads[JsObject] = {
-    (
-      (__ \ Symbol("eventType")).json.put(JsString(s"Event$Event8A")) and
+   (
+      (pathToEvent \ Symbol("eventType")).json.put(JsString(s"Event$Event8A")) and
         readsIndividualMemberDetails and
         (pathPaymentDetails \ Symbol("reasonBenefitTaken")).json.copyFrom(readsPaymentTypeEvent8A) and
         (pathPaymentDetails \ Symbol("typeOfProtection")).json.copyFrom(readsTypeOfProtectionEvent8A.orElse((Reads.pure(JsString("N/A"))))) and
         (pathPaymentDetails \ Symbol("amountLumpSum")).json.copyFrom((pathLumpSumAmountAndDateDetails \ Symbol("lumpSumAmount")).json.pick) and
         (pathPaymentDetails \ Symbol("eventDate")).json.copyFrom((pathLumpSumAmountAndDateDetails \ Symbol("lumpSumDate")).json.pick) and
-        (pathPaymentDetails \ Symbol("freeText")).json.copyFrom(readsTypeOfProtectionReferenceEvent8A.orElse(Reads.pure(JsString("N/A"))))).reduce
+        (pathPaymentDetails \ Symbol("freeText")).json.copyFrom(readsTypeOfProtectionReferenceEvent8A.orElse(Reads.pure(JsString("N/A")))) and
```

Comment:
Same as previous

Reply:
i THINK for all these comments the todo change will be on a separate tickedt anyway so i wouldn't bother